source/how-tos/support_ticket.rst

```diff
@@ -0,0 +1,132 @@
+  For more information see the `Rails documentation on mailers <https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration>`_.
+* ``delivery_settings``: Rails settings object to set the transport configuration properties.
+  See example below or the `Rails documentation on mailers <https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration>`_.
+
+Sample configuration:
+
+  .. code-block:: yaml
+
+    support_ticket:
+      email:
+        # Optional.
+        # Text/HTML added to the page under the support ticket header
+        description: |
+          <strong>My Description Text</strong><br />
+          This is a description text for the frontend with HTML support
+        # Optional.
+        # To set the email from field. Defaults to the email set in the form.
+        from: "config@example.com" 
+        # Required.
+        # Your support system email address. The email will be sent to this address
+        to: "support@support.system.com"
+        # delivery_method and delivery_settings are Rails configuration items
+        # that can be set in this file to simplify configuration.
+        # If you prefer, you can configure Rails in the usual way.
+        delivery_method: "smtp"
+        delivery_settings:
+          address: 'smtp.gmail.com'
+          port: 587
+          domain: 'example.com'
+          user_name: '<username>'
+          password: '<password>'
+          authentication: 'plain'
+          enable_starttls_auto: true
+          open_timeout: 5
+          read_timeout: 5
+
+Customizations
+..............
+
+The support ticket form can be customized using the ``form`` and ``attributes`` configuration properties.
+In the same way that you can configure the :ref:`form for interactive applications <enable-desktops-modify-form-attributes>`,
+the support ticket form fields can be arranged or changed as required.
+
+.. note::
+  
+  Please note that an ``email`` field must be present in the form for the support ticket feature to be funtional.
+
+Custom fields can be added, but they will require a custom email template to make use of the field values to generate the email body.
+Drop the template file into ``/etc/ood/config/apps/dashboard/views/support_ticket/email`` and reference it
+in the configuration. This template file can be any format Rails recognizes for emails, usually the ``.txt.erb`` extension.
+
+The example below shows a custom template configuration with 3 fields and how to use them in the template to generate the email body.
+
+  .. code-block:: yaml
+
+    support_ticket:
+      email_template: "custom_template"
+      attributes:
+        email:
+          required: true
+        subject:
+          value: "OnDemand Problem: "
+      form:
+        - email
+        - subject
+        - department
+
+  .. code-block::
+
+    # /etc/ood/config/apps/dashboard/views/support_ticket/email/custom_template.txt.erb
```

Comment:
```suggestion
    # /etc/ood/config/apps/dashboard/views/support_ticket/email/_custom_template.txt.erb
```

Is this right?

Reply:
That is correct. For email templates, Rails does not require the `_`.
I tested the configuration while doing the documentation.

Reply:
Oh because we pull it, not because it's a part of the rails infrastructure. 

Hmmmmm, this is different than all the other partials they can overload then.  I wonder if we should force the prepended `_` so that it's like every other partial.

Reply:
I think we can use partials with email templates too. We could create en email layout template and include the default or custom as partials. This will follow the same structure as the rest of the template overrides in OnDemand.

In this case, the email layout template will be empty, just with the render partial statement.

Reply:
Yes we can use partials. I will update to use partials and update the docs.
This will make consistent with everything else

Reply:
Completed